src/app.ts

```diff
@@ -1,4 +1,10 @@
-console.log('hellod boyy');
+import {Cluster, Itur, Constitution} from './types';
+import {readClusters, readIturim, readConstitution} from './dataProviders/csvProvider';
 
-const mysum: String = 'uri';
-console.log(mysum);
+const processData = async () => {
```

Comment:
Since this function mainly read things I think a better name could be readData

Reply:
Done

Reply:
Done